[PROCAPPS-858] Update Container Image Docs to include Trends Page (#22794)
DataDog/documentation: content/en/infrastructure/containers/container_images.md

{{< img src="security/vulnerabilities/container_images.png" alt="The container images view highlighting vulnerabilities and container column sort feature" width="100%">}}

> The [container image trends view][9] provides high-level insights across all of your images in your containerized infrastructure. Container Image Trends metrics can help you answer key questions about your security posture and deployment footprint over the span of weeks and months.

Reviewer: ```suggestion
The [container image trends view][9] provides high-level insights across all of your images in your containerized infrastructure. Container image trends metrics can help you answer key questions about your security posture and deployment footprint over the span of weeks and months.
```

{{< img src="security/vulnerabilities/container_images.png" alt="The container images view highlighting vulnerabilities and container column sort feature" width="100%">}}

> The [container image trends view][9] provides high-level insights across all of your images in your containerized infrastructure. Container Image Trends metrics can help you answer key questions about your image posture and deployment footprint over the span of weeks and months.

Reviewer 2: +1

Set up the [AWS integration][4] to begin crawling Container Image metadata from Amazon ECR.

## Configure container images trends

> Use the Container Image Trends Configuration modal and toggle 'Enable Container Image Metric Collection' to turn on the generation of image metrics.

Reviewer: ```suggestion
Use the container image trends configuration modal and toggle **Enable Container Image Metric Collection** to turn on image metric generation.
```

> Image metrics are collected from the [Live Containers](#live-containers) and [Image Check](#image-collection) sources. Follow the same instructions above to ensure that these collections are enabled across your entire infrastructure and take full advantage of the Trends view.

Reviewer: ```suggestion
Image metrics are collected from the [Live Containers](#live-containers) and [Image Check](#image-collection) sources. Follow the same instructions as above to ensure that these collections are enabled across your entire infrastructure and take full advantage of the trends view.
```

Set up the [AWS integration][4] to begin crawling Container Image metadata from Amazon ECR.

> ## Configure container images trends view

Author: Tried to make the title similar to the `## Configure container images view` in ln 29, but not too opinionated on this

> Image metrics on the Container Image Trends view are collected from the Live Containers and Image Collection source. The same instructions above allow the collection of container image metrics.

Reviewer 2: ```suggestion
Live Containers and Image Check sources
```

> Image metrics on the Container Image Trends view are collected from the Live Containers and Image Collection source. The same instructions above allow the collection of container image metrics.

Reviewer 2: ```suggestion
Image metrics on the Container Image Trends view are collected from the Live Containers and Image Check sources. The same instructions above allow the collection of container image metrics.
```
